src/components/molecules/TableFooter/TableFooter.tsx

```diff
@@ -14,14 +16,13 @@ const TableFooter: React.FC<TableFooterProps> = ({
       <div className="action">
         <div className="focus">Focus on:</div>
         <div className="focus-option">Jazz</div>
-        <label className="switch">
-          <input
-            type="checkbox"
-            checked={!isVideoFocused}
-            onChange={() => setIsVideoFocused(!isVideoFocused)}
-          />
-          <span className="slider" />
-        </label>
+        <Toggler
+          labelClassName="switch"
+          id="showRadio"
+          name="showRadio"
+          defaultToggled={!isVideoFocused}
+          onToggle={() => setIsVideoFocused(!isVideoFocused)}
+        />
```

Comment:
It looks like it was a controlled component and you made it not controlled. Am I missing something? It used to have `checked` prop, which was turned into `defaultToggled/defaultChecked`

Reply:
It should have no labelClassName, the label should be passed directly as a prop

Reply:
Please, apply this knowledge to all of the other occurences^

Reply:
My knowledge of this component is limited, the `Toggler` props as well have two of them required:
```
export interface TogglerProps extends CheckboxProps {
  defaultToggled: boolean;
  onToggle: ChangeEventHandler<HTMLInputElement>;
  containerClassName?: string;
}
```
Upon putting `checked` instead of `defaultToggled` there is console error and component stops functioning.

Reply:
![toggler-issues-02](https://user-images.githubusercontent.com/79229621/125104981-40d51500-e0de-11eb-87ad-2391a545c202.png)
![toggler-issues-01](https://user-images.githubusercontent.com/79229621/125104987-429ed880-e0de-11eb-81f0-7319e45b86b2.png)

Reply:
made `Toggler` props optional in 7983b12912a95f067c32c1fc6ec8d47716e372d0 and b61a17d078f2d76550c6d8cf8157c86ce2f13385 and removed the `labelClassName`

Reply:
FYI: @mike-lvov the API of that toggled component is super restrictive currently. It doesn’t allow us to pass through things like register() from react form hooks, etc.

Reply:
@0xdevalias At the suggestion of @mike-lvov  I have removed restrictions in 7983b12912a95f067c32c1fc6ec8d47716e372d0 and 7983b12912a95f067c32c1fc6ec8d47716e372d0.

Also in 3c68fdfc66a9346cd67e1d15bd5fe7a688d52090, I have made  `name` prop to default to `id` if the former isn't set but the latter is. This way there isn't repetition of both props in case some calculation is done or a typo makes them different.

Reply:
Removed uses of `labelClassName` https://github.com/sparkletown/sparkle/pull/1775/commits/7983b12912a95f067c32c1fc6ec8d47716e372d0

Reply:
Actually, looking at this again it's extending from `CheckboxProps`, so not as restrictive as I was thinking it was (I always miss the `extends` in these..)

@goran-peoski-work will review once i'm done with the other PR. I'm not 100% sure the changes made here are correct. But will need to look at them together to make that call for certain.

Reply:
See my review in https://github.com/sparkletown/sparkle/pull/1775#pullrequestreview-703491797

Marking this resolved again.